src/web-ui/.env

```diff
@@ -1,27 +1,27 @@
-# This file is automatically generated by gen_env.sh during the build stage 
-# in CodePipeline. It is also used directly when running the web-ui locally 
-# in a Docker container. When running the web-ui locally, update the values 
+# This file is automatically generated by gen_env.sh during the build stage
+# in CodePipeline. It is also used directly when running the web-ui locally
+# in a Docker container. When running the web-ui locally, update the values
 # below to match your configuration.
 
 # Configure Dependent Services
-VUE_APP_PRODUCTS_SERVICE_DOMAIN=http://localhost
-VUE_APP_PRODUCTS_SERVICE_PORT=8001
-VUE_APP_USERS_SERVICE_DOMAIN=http://localhost
-VUE_APP_USERS_SERVICE_PORT=8002
-VUE_APP_CARTS_SERVICE_DOMAIN=http://localhost
-VUE_APP_CARTS_SERVICE_PORT=8003
-VUE_APP_ORDERS_SERVICE_DOMAIN=http://localhost
-VUE_APP_ORDERS_SERVICE_PORT=8004
-VUE_APP_RECOMMENDATIONS_SERVICE_DOMAIN=http://localhost
-VUE_APP_RECOMMENDATIONS_SERVICE_PORT=8005
-VUE_APP_SEARCH_SERVICE_DOMAIN=http://localhost
-VUE_APP_SEARCH_SERVICE_PORT=9200
+VUE_APP_PRODUCTS_SERVICE_DOMAIN=http://retai-LoadB-180OKYDHA0LYM-1573441342.us-west-2.elb.amazonaws.com
+VUE_APP_PRODUCTS_SERVICE_PORT=80
+VUE_APP_USERS_SERVICE_DOMAIN=http://retai-LoadB-9L56IAAMLLO9-80589350.us-west-2.elb.amazonaws.com
+VUE_APP_USERS_SERVICE_PORT=80
+VUE_APP_CARTS_SERVICE_DOMAIN=http://retai-LoadB-6WZ1OVEQCZAO-98388682.us-west-2.elb.amazonaws.com
+VUE_APP_CARTS_SERVICE_PORT=80
+VUE_APP_ORDERS_SERVICE_DOMAIN=http://retai-LoadB-X3NLTVMFI7SP-114615757.us-west-2.elb.amazonaws.com
+VUE_APP_ORDERS_SERVICE_PORT=80
+VUE_APP_RECOMMENDATIONS_SERVICE_DOMAIN=http://retai-LoadB-PPAUX0RMQFLN-1947944783.us-west-2.elb.amazonaws.com
+VUE_APP_RECOMMENDATIONS_SERVICE_PORT=80
+VUE_APP_SEARCH_SERVICE_DOMAIN=http://retai-LoadB-GH3PHNM4AQU2-1006238650.us-west-2.elb.amazonaws.com
+VUE_APP_SEARCH_SERVICE_PORT=80
 
 # Configure Authentication
 VUE_APP_AWS_REGION=us-west-2
-VUE_APP_AWS_IDENTITY_POOL_ID=
-VUE_APP_AWS_USER_POOL_ID=
-VUE_APP_AWS_USER_POOL_CLIENT_ID=
+VUE_APP_AWS_IDENTITY_POOL_ID=us-west-2:47830704-39b5-4258-94f8-cb0efeed73dd
+VUE_APP_AWS_USER_POOL_ID=us-west-2_q4epyIDJp
+VUE_APP_AWS_USER_POOL_CLIENT_ID=1u5em88069h1fphhtti77e2qdb
```

Comment:
Do not check these changes in. Remove your updates to this file but keep the ones from the merge.

Reply:
Oops. I missed to remove this file. Sorry about that.